src/Data/Array.purs

```diff
@@ -71,6 +71,8 @@ module Data.Array
   , mapMaybe
   , catMaybes
   , mapWithIndex
+  , updateAtIndices
```

Comment:
Perhaps put these next to `updateAt` and `modifyAt` in the export list, so that when we (I) eventually get around to ordering declarations in HTML docs based on their order of appearance in the export list the docs read better?